src/css/custom.css

```diff
@@ -62,12 +64,16 @@ html[data-theme='dark'] .navbar-item-github:before {
   background-position: center;
 }
 
-@media screen and (max-width: 996px) {
+@media screen and (max-width: 1024px) {
   .navbar-item-github {
     display: none !important;
   }
 }
 
+.navbar__link {
+  text-align: center;
+}
```

Comment:
When the nav items wrap, it looked weird not centered. Centering the items makes the spacing look more consistent